src/main/scala/io/iohk/ethereum/domain/Blockchain.scala

```diff
@@ -456,18 +473,20 @@ class BlockchainImpl(
     )
 
   //FIXME Maybe we can use this one in regular execution too and persist underlying storage when block execution is successful
```

Comment:
maybe create ticket for this `FIXME` ? as it seems having two separate methods to retrieve execution world was a bit error prone